index.js

```diff
@@ -34,6 +36,11 @@ function findWorkspaceRoot(initial) {
   return null;
 }
 
+function extractWorkspaces(manifest) {
+  const workspaces = (manifest || {}).workspaces;
+  return (workspaces && workspaces.packages) || (workspaces instanceof Array ? workspaces : null);
```

Comment:
Should prefer `Array.isArray(workspaces)`.

Reply:
I guess this will break the CI Node.js 4 task too

Reply:
It shouldn't…

```
> process.version
'v4.8.7'
> Array.isArray
[Function: isArray]
```